Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -186,6 +186,10 @@ class LoginListViewController: SensitiveViewController {
 
 // MARK: - Selectors
 private extension LoginListViewController {
+    @objc func remoteLoginsDidChange() {
+        loadLogins()
+    }
+
     @objc func loadLogins(query: String? = nil) {
```

Comment:
if this is no longer called as a selector from anywhere (which I don't think it is) then we can remove the `@objc` decorator.